src/IsraeliTMRef.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PHPCoord;
+
+/**
+ * Irish Transverse Mercator grid ref (the new Irish Grid system)
+ * References are accurate to 1m.
+ * @author Doug Wright
+ */
+class IsraeliTMRef extends TransverseMercator
+{
+    /**
+     * @return RefEll
+     */
+    public function getReferenceEllipsoid(): RefEll
+    {
+        return RefEll::grs80();
+    }
+
+    /**
+     * @return float
+     */
+    public function getScaleFactor(): float
+    {
+        return 1.0000067;
+    }
+
+    /**
+     * @return int
+     */
+    public function getOriginNorthing(): int
+    {
+        return 631556;
```

Comment:
Do you have a source for this value? (and for the false easting). I note that Wikipedia is giving different values (https://en.wikipedia.org/wiki/Israeli_Transverse_Mercator) which makes me nervous